alibi_detect/od/tests/test_ensemble.py

```diff
@@ -0,0 +1,161 @@
+import pytest
+import torch
+
+from alibi_detect.od.pytorch import ensemble
+from alibi_detect.exceptions import NotFittedError
+
+
+def test_pval_normalizer():
```

Comment:
These tests are fairly self explanatory IMO, but just noting that they are the only new ones without docstrings...